src/js/fozzie.js

```diff
@@ -1,7 +0,0 @@
-import module from './module';
-
-const fozzie = () => 'Wocka wocka wocka!';
-
-module();
```

Comment:
Is it worth keeping this in for is we want to move some of the more common JS into the base of fozzie?

Reply:
I think it's OK for now. 

I've created a related issue (#30) where I've linked to the commit which removed the JS for reference when we come to add some JS back in.